diff --git a/CMakeModules/BundleTarget.cmake b/CMakeModules/BundleTarget.cmake
index bd30faf91..6d00a0e42 100644
--- a/CMakeModules/BundleTarget.cmake
+++ b/CMakeModules/BundleTarget.cmake
@@ -119,6 +119,7 @@ if (BUNDLE_TARGET_EXECUTE)
 
         # Set up app icon
         file(COPY_FILE "${source_path}/dist/azahar.svg" "${CMAKE_BINARY_DIR}/dist/org.azahar_emu.Azahar.svg")
+        file(COPY_FILE "${source_path}/dist/azahar.desktop" "${CMAKE_BINARY_DIR}/dist/org.azahar_emu.Azahar.desktop")
 
         message(STATUS "Creating AppDir for executable ${executable_path}")
         execute_process(COMMAND ${CMAKE_COMMAND} -E env
@@ -128,7 +129,7 @@ if (BUNDLE_TARGET_EXECUTE)
             --plugin checkrt
             --executable "${executable_path}"
             --icon-file "${CMAKE_BINARY_DIR}/dist/org.azahar_emu.Azahar.svg"
-            --desktop-file "${source_path}/dist/${executable_name}.desktop"
+            --desktop-file "${CMAKE_BINARY_DIR}/dist/org.azahar_emu.Azahar.desktop"
             --appdir "${appdir_path}"
             RESULT_VARIABLE linuxdeploy_appdir_result)
         if (NOT linuxdeploy_appdir_result EQUAL "0")
