diff --git a/src/android/app/src/main/java/org/citra/citra_emu/fragments/HomeSettingsFragment.kt b/src/android/app/src/main/java/org/citra/citra_emu/fragments/HomeSettingsFragment.kt
index 432a06aa0d..e7ff2ba341 100644
--- a/src/android/app/src/main/java/org/citra/citra_emu/fragments/HomeSettingsFragment.kt
+++ b/src/android/app/src/main/java/org/citra/citra_emu/fragments/HomeSettingsFragment.kt
@@ -1,4 +1,4 @@
-// Copyright Citra Emulator Project / Azahar Emulator Project
+// Copyright 2023 Citra Emulator Project
 // Licensed under GPLv2 or any later version
 // Refer to the license.txt file included.

@@ -121,14 +121,8 @@ class HomeSettingsFragment : Fragment() {
                 }
             ),
             HomeSetting(
-                R.string.install_game_content,
-                R.string.install_game_content_description,
-                R.drawable.ic_install,
-                { mainActivity.ciaFileInstaller.launch(true) }
-            ),
-            HomeSetting(
-                R.string.setup_system_files,
-                R.string.setup_system_files_description,
+                R.string.system_files,
+                R.string.system_files_description,
                 R.drawable.ic_system_update,
                 {
                     exitTransition = MaterialSharedAxis(MaterialSharedAxis.X, true)
@@ -136,6 +130,12 @@ class HomeSettingsFragment : Fragment() {
                         ?.navigate(R.id.action_homeSettingsFragment_to_systemFilesFragment)
                 }
             ),
+            HomeSetting(
+                R.string.install_game_content,
+                R.string.install_game_content_description,
+                R.drawable.ic_install,
+                { mainActivity.ciaFileInstaller.launch(true) }
+            ),
             HomeSetting(
                 R.string.share_log,
                 R.string.share_log_description,
