diff --git a/src/core/file_sys/ticket.h b/src/core/file_sys/ticket.h
index db97cf09f..efeeec76f 100644
--- a/src/core/file_sys/ticket.h
+++ b/src/core/file_sys/ticket.h
@@ -49,8 +49,9 @@ public:
         u8 audit;
         INSERT_PADDING_BYTES(0x42);
         std::array<LimitEntry, 0x8> limits;
+        std::array<u8, 0xAC> content_index;
     };
-    static_assert(sizeof(Body) == 0x164, "Ticket body structure size is wrong");
+    static_assert(sizeof(Body) == 0x210, "Ticket body structure size is wrong");
 #pragma pack(pop)

     Loader::ResultStatus DoTitlekeyFixup();
