diff --git a/src/android/app/build.gradle.kts b/src/android/app/build.gradle.kts
index 25e214441..4efa27b57 100644
--- a/src/android/app/build.gradle.kts
+++ b/src/android/app/build.gradle.kts
@@ -61,9 +61,7 @@ android {
     }
 
     defaultConfig {
-        // The application ID refers to Lime3DS to allow for
-        // the Play Store listing, which was originally set up for Lime3DS, to still be used.
-        applicationId = "io.github.lime3ds.android"
+        applicationId = "io.github.azahar.android"
         minSdk = 28
         targetSdk = 35
         versionCode = autoVersion
